galasa-extensions-parent/dev.galasa.ecosystem.docker.builder/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+
+  	<parent>
+  		<groupId>dev.galasa</groupId>
+  		<artifactId>galasa-extensions-parent</artifactId>
+  		<version>0.7.0-SNAPSHOT</version>
+  	</parent>
+  		
+  		<artifactId>dev.galasa.ecosystem.docker.builder</artifactId>
+  		<version>0.7.0-SNAPSHOT</version>
+  		<packaging>jar</packaging>
+  		<name>Galasa Ecosystem Docker Builder</name>
+  		<description>Builds Galasa Ecosystem using default or user defined properties for docker</description>
+  		
+  		  <properties>
+   		    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+          </properties>
+  		
+  		
+  		<dependencies> 
+  		  <dependency>
+    	    <groupId>org.yaml</groupId>
+    	    <artifactId>snakeyaml</artifactId>
+    	    <version>1.25</version>
+    	    <type>jar</type>
+  		  </dependency>
+  		  <dependency>
+  		    <groupId>org.apache.maven.plugins</groupId>
+  		    <artifactId>maven-jar-plugin</artifactId>
+  		    <version>2.4</version>
+  		  </dependency>
+  		</dependencies>
+  		
+  		
+  <build>
+   <plugins>
+   <plugin>
+     <groupId>org.apache.maven.plugins</groupId>
+     <artifactId>maven-dependency-plugin</artifactId>
+     <executions>
+      <execution>
+       <id>copy-dependencies</id>
```

Comment:
don't use "copy-dependencies" it will not work well when running as a JAR.   Instead "shade" in the snake-yaml jar